bundle/internal/schema/main.go

```diff
@@ -66,6 +68,24 @@ func addInterpolationPatterns(typ reflect.Type, s jsonschema.Schema) jsonschema.
 	}
 }
 
+func removeDeprecatedJobsFields(typ reflect.Type, s jsonschema.Schema) jsonschema.Schema {
+	switch typ {
+	case reflect.TypeOf(resources.Job{}):
+		delete(s.Properties, "deployment")
+		delete(s.Properties, "edit_mode")
+		delete(s.Properties, "format")
```

Comment:
Please include a comment for each of these, saying if it is 1) deprecated, 2) intentionally hidden, or 3) read-only.

This is too opaque.

Reply:
done